Makefile

```diff
@@ -0,0 +1,53 @@
+export CQ_PROVIDER_DEBUG=1
+export CQ_REATTACH_PROVIDERS=.cq_reattach
+
+# install the latest version of CQ
+install-cq:
+	curl -L https://github.com/cloudquery/cloudquery/releases/latest/download/cloudquery_Darwin_x86_64 -o cloudquery
+	chmod +x ./cloudquery
+
+install-cq-mac:
+	curl -L https://github.com/cloudquery/cloudquery/releases/latest/download/cloudquery_Darwin_x86_64 -o cloudquery
+	chmod +x ./cloudquery
+
+install-cq-mac:
+	curl -L https://github.com/cloudquery/cloudquery/releases/latest/download/cloudquery_Darwin_x86_64 -o cloudquery
+	chmod +x ./cloudquery
+
+
+# build the cq aws provider
+build:
+	go build -o cq-provider-aws
+
+# build and run the cq aws provider
+run: build
```

Comment:
maybe just `go run main.go`

Reply:
So I had it like this because originally I was passing in flags in the build stage and wanted to only have to maintain them in a single place